features/step_definitions/posts_steps.rb

```diff
@@ -27,6 +27,13 @@
   user.post(:status_message, :text => text, :public => true, :to => user.aspect_ids)
 end
 
+Given /^there are (\d+) public posts from "([^"]*)"$/ do |n_posts, email|
+  user = User.find_by_email(email)
+  (1..n_posts.to_i).each do |n|
+    user.post(:status_message, :text => 'post nr. ' + n.to_s, :public => true, :to => user.aspect_ids)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.